gallery/src/components/NxTextLink/NxTextLinkExternalExample.tsx

```diff
@@ -6,10 +6,18 @@
  */
 import React from 'react';
 
-import { NxTextLink, NxP } from '@sonatype/react-shared-components';
+import { NxCode, NxTextLink, NxP } from '@sonatype/react-shared-components';
 
 export default () =>
-  <NxP>
-    The quick brown fox <NxTextLink external href="https://html.spec.whatwg.org/multipage/">MDN</NxTextLink> jumped
-    over the lazy dog.
-  </NxP>;
+  <>
+    <NxP>
+      The quick brown fox <NxTextLink external href="https://html.spec.whatwg.org/multipage/">MDN</NxTextLink> jumped
+      over the lazy dog.
+    </NxP>
+    <NxP>
+      Learn more about{' '}
+      <NxCode>
+        <NxTextLink external href="https://www.a11yproject.com/">a11y</NxTextLink>
```

Comment:
Good thinking including this case, however I think it definitely needs to be checked with design.  I strongly suspect that how it is right now isn't what they'll desire, as the alignment between the icon and the background doesn't look good.
![image](https://user-images.githubusercontent.com/3630091/143900957-5d79782e-089c-4012-bcbd-37a5b74cda3e.png)

Reply:
Good thinking including this case, however I think it definitely needs to be checked with design. I strongly suspect that how it is right now isn't what they'll desire, as the alignment between the icon and the background doesn't look good.

The icon alignment have been fixed here: 41dd3cf609d38fc97bafb710a1a1c6007289155a
Still working on alignment with design on updating Kaleido but we're very close.